plugins/show-language/prism-show-language.js

```diff
@@ -1,12 +1,10 @@
 (function () {
-
-	if (typeof self === 'undefined' || !self.Prism || !self.document) {
+	if (typeof self === "undefined" || !self.Prism || !self.document) {
 		return;
 	}
 
 	if (!Prism.plugins.toolbar) {
-		console.warn('Show Languages plugin loaded before Toolbar plugin.');
-
```

Comment:
Please revert these changes. While the file might need a cleanup, it shouldn't be part of this PR.

The only change to this file should be this:

```diff
 		"ichigojam": "IchigoJam",
+		"iecst": "Structured Text (IEC 61131-3)",
 		"inform7": "Inform 7",
```

Reply:
How? when I run npx gulp it rebuild this file and make them all one line.

Reply:
And yet, I am not sure how to revert a single file change. Out of my knowledge. I'll try to google it.

Reply:
The easiest way would probably to copy-paste the content of the current [`prism-show-language.js`](https://github.com/PrismJS/prism/blob/master/plugins/show-language/prism-show-language.js) in master and run `npx gulp`.

There's probably also some fancy git-way of doing it, but I'm not a git expert...